docs/dev/table/connectors/formats/maxwell.md

```diff
@@ -152,8 +152,9 @@ Format Options
       <td>optional</td>
       <td style="word-wrap: break-word;">false</td>
       <td>Boolean</td>
-      <td>Skip fields and rows with parse errors instead of failing.
-      Fields are set to null in case of errors.</td>
+      <td>Skip parse errors instead of failing. 
+      This message will be ignored and returned <code>null</code> when deserializing the <code>byte[]</code> to the <code>JsonNode</code> fails.(e.g the message with invalid json format, the message contains unescaped control characters) 
+      Otherwise fields are set to null in case of errors.</td>
```

Comment:
This is very confusing. Does it skip the message or return `null` or set field to `null`???